app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    parts = command.split()
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file == destination_file:
```

Comment:
You should also check that the command is actually `"cp"` here, in case not, you could return.